mongodb/go.mod

```diff
@@ -1,24 +1,70 @@
 module github.com/gofiber/storage/mongodb/v2
 
-go 1.19
+go 1.21
+
+toolchain go1.22.1
```

Comment:
The toolchain needs to be removed

Reply:
This is why mongo tests are failing

Reply:
Sorry about that, wrong go mod tidy + commit